Dockerfile

```diff
@@ -4,7 +4,7 @@ RUN apt-get update --fix-missing
 RUN apt-get install -y apt-transport-https
 RUN apt-get install -y tar gzip zip wget curl 
 RUN apt-get install -y build-essential software-properties-common
-RUN apt-get install -y python3 python3-pip git-all
+RUN apt-get install -y python3.8 python3-pip git-all
```

Comment:
I wonder if we could push the Python version into a single point of change? It might not be worth it just for Python, to be fair.